website/content/en/preview/getting-started/migrating-from-cas/scripts/step08-generate-chart.sh

```diff
@@ -0,0 +1,7 @@
+helm template --namespace karpenter \
+    karpenter karpenter/karpenter \
+    --set aws.defaultInstanceProfile=KarpenterInstanceNodeRole \
+    --set clusterEndpoint="${OIDC_ENDPOINT}" \
```

Comment:
@rothgar @bwagner5 Should this be `CLUSTER_ENDPOINT` instead of `OIDC_ENDPOINT`. All other methods of getting started use `CLUSTER_ENDPOINT`: 
* [getting-started-with-eksctl](https://github.com/aws/karpenter/blob/befc00c890878c6b531e7c07acd7be1d4a92f6ec/website/content/en/preview/getting-started/getting-started-with-eksctl/scripts/step08-apply-helm-chart.sh)

* [getting-started-with-terraform](https://github.com/aws/karpenter/blame/main/website/content/en/preview/getting-started/getting-started-with-terraform/_index.md#L291) 

I skimmed through the [testing video on twitch](https://www.twitch.tv/videos/1448421517?t=00h27m13s)  (so, I might have missed something), but Karpenter was setup using `getting-started-with-eksctl` instructions. See video from 00:27:13 - 00:28:25.